_layouts/layout.html

```diff
@@ -82,6 +82,12 @@
     </script>
     {% endif %}
 
+      
+
+  <!-- Used for LATEX equations -->
+<script type="text/javascript" src="http://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>
+<script type="text/javascript" src="/js/jq_mathjax_parse.js"></script>
```

Comment:
Are you sure we need this?  We already have working mathjax support, see eg [this post](http://gallery.rcpp.org/articles/EM-algorithm-example/).

@jjallaire Any comments?